### ISSUE TITLE
Failed prop type: Invalid prop `placeholderTextColor` supplied to `Input`

### ISSUE THREAD

**Issue by zhGio** [tamagui/tamagui#1111]
### Current Behavior

Running tamagui on react-native & expo, latest versions, and in my `tamagui.config.ts` i have the colors defined for both themes, i.e. 
`  textSofter: "#7c7e91",`, and they get applied and everything works in runtime, but in compile time i keep getting the error message

```
 ERROR  Warning: Failed prop type: Invalid prop `placeholderTextColor` supplied to `Input`: $textSofter
Valid color formats are
  - '#f0f' (#rgb)
  - '#f0fc' (#rgba)
  - '#ff00ff' (#rrggbb)
  - '#ff00ff00' (#rrggbbaa)
  - 'rgb(255, 255, 255)'
  - 'rgba(255, 255, 255, 1.0)'
  - 'hsl(360, 100%, 100%)'
  - 'hsla(360, 100%, 100%, 1.0)'
  - 'transparent'
  - 'red'
  - 0xff00ff00 (0xrrggbbaa)
```

```
import { Input } from "tamagui";

...
      <Input
        bg="$backgroundSoft"
        placeholderTextColor="$textSofter"
... />
```

### Expected Behavior

Not to see the error message in console.

### Tamagui Version

```markdown
1.23.0 but i can confirm that it happened to all my previous versions which i tried upgrading in order to resolve this message, ever since 1.18 or so..
```

### Reproduction

```markdown
Cannot run the expo sandbox, but this should be easily reproducible..
```

**Comment by natew** [comment]
fix in `v1.30.8`

**Comment by rizikolik** [comment]
This is happening at version 1.91.3
